Client/Application/AppDelegate.swift

```diff
@@ -441,6 +442,15 @@ extension AppDelegate: MFMailComposeViewControllerDelegate {
     }
 }
 
+extension AppDelegate: BrowserViewControllerDelegate {
+    func browserViewControllerAddReleaseNotesOnUpdate(browserViewController: BrowserViewController, profile: Profile) {
+        if profile.prefs.stringForKey("latestAppVersion") != appVersion {
+            browserViewController.openURLInNewTab(NSURL(string: "https://www.mozilla.org/en-US/firefox/ios/\(appVersion)/releasenotes/")!)
+        }
+        profile.prefs.setString(appVersion, forKey: "latestAppVersion")
+    }
+}
```

Comment:
This isn't great -- we're making assumptions about the state of `BVC` (a view) to decide when to add a new tab (our model). Ideally, we'd tease apart the coupling of adding a tab and inserting the web view. This might be OK for now until we can do our BVC refactor.